pkgs/misc/emulators/oberon-risc-emu/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, SDL2 }:
+
+stdenv.mkDerivation {
+  pname = "oberon-risc-emu";
+  version = "unstable-2020-08-18";
```

`unstable` should be apart of `pname`, not version. See https://github.com/NixOS/nixpkgs/issues/68518
```suggestion
  pname = "oberon-risc-emu-unstable";
  version = "2020-08-18";
```